elodie.py

```diff
@@ -110,6 +112,9 @@ def _import(destination, source, file, album_from_folder, trash, allow_duplicate
 
     files = set()
     paths = set(paths)
+    excludeDir = set()
+    if exclude:
+        excludeDir.update(exclude)
```

Comment:
Instead of excluding a directory can we take in a regular expression to be matched against the file path? I think then we won't need to use a `set()` since it looks that only let you use `in`.

Reply:
I use `set()` to accept optional multiple exclusions. Is there another way to do it? Please remember I am a newbie. 😄 And pointing me to how to add tests would be appreciated.

Reply:
Got it. I didn't realize `multiple=True` in the `click` decorator does this. Looks good.